runtime/compiler/optimizer/VectorAPIExpansion.cpp

```diff
@@ -1030,7 +1091,8 @@ TR::Node *TR_VectorAPIExpansion::addHandler(TR_VectorAPIExpansion *opt, TR::Tree
    TR::Compilation *comp = opt->comp();
 
    if (mode == checkScalarization) return node;
-   if (mode == checkVectorization) return NULL;  // TODO: check type and platform
+   if (mode == checkVectorization)
+      return supportedOnPlatform(comp, vectorLength) ? node : NULL; // TODO: check type and platform
```

Comment:
I added another comment and actually disabled that handler since it's not really needed. It was from the times when we wanted to recognize high level methods. But I would like to keep it for now.